src/config/periods.js

```diff
@@ -7,7 +7,8 @@ import {
 } from 'date-fns';
 import i18n from '@/locale/i18n';
 
-export const DEFAULT_PERIOD = 'quarter';
+export const DEFAULT_OKR_PERIOD = 'quarter';
+export const DEFAULT_KPI_PERIOD = 'all';
```

Comment:
Litt usikker på om dette gir mest mening som standard, vil tro at kanskje "i år" eller "siste 6 mnd" vil gi mer mening for de fleste? Ta en felles diskusjon på det?

Reply:
Ja, vurderte "hittil i år" også - så kan gjerne bli enige alle - se litt på hvordan det blir på ekte data, og ellers hva som gir mening 👍